README.md

```diff
@@ -53,6 +53,8 @@ ln.finish()  # finish the run
 
 <!-- from here on, slight deviation from preface.md, where all this is treated in the walk through in more depth -->
 
+The code snippet above creates an artifact in your instance. Artifacts represent datasets and models stored as files or folders in various formats.
+
 Running this code inside a script via `python create-fasta.py` produces the following data lineage.
```

Comment:
Because you're introducing "code snippet", "Running this code" then feels off. Can you either say just "code" above or adapt line 58 to say "Running it"...?

Reply:
done